devicetypes/smartthings/eaton-5-scene-keypad.src/eaton-5-scene-keypad.groovy

```diff
@@ -12,7 +12,7 @@
  *
  */
 metadata {
-	definition(name: "Eaton 5-Scene Keypad", namespace: "smartthings", author: "SmartThings", mcdSync: true) {
+	definition(name: "Eaton 5-Scene Keypad", namespace: "smartthings", author: "SmartThings", mcdSync: true, mnmn: "SmartThings", vid: "SmartThings-smartthings-Eaton_5-Scene_Keypad") {
```

Comment:
Is the explicit vid necessary? I just noticed the metadata wasn't in production, so the client was definitely using the default generic. Was there a problem with the correct vid being generated when metadata is staging? (I just want to make sure that there isn't an underlying problem where this doesn't work without an explicit mnmn and vid that might be affecting other devices, too.)

Reply:
There's an older one that is `5_Scene` instead of `5-Scene`. I asked Bashi to test first without the explicit vid but it wasn't getting picked up.